beta-src/src/models/Interfaces.ts

```diff
@@ -0,0 +1,64 @@
+export interface IBoard {
+  territories: ITerritory[];
+  units: IUnit[];
+  myUnits: IUnit[];
+  terrStatus: ITerrStatus[];
+  context: IContext;
+}
+
+export interface ITerritory {
+  id: number;
+  name: string;
+  type: string;
+  supply: string;
+  countryID: number;
+  coast: string;
+  coastParentID: number;
+  smallMapX: number;
+  smallMapY: number;
+  Borders: Array<IBorder>;
+  CoastalBorders: Array<ICoastalBorder>;
+  coastParent: ITerritory;
+  Unit: IUnit;
+  unitID: number;
+}
+
+export interface IBorder {
+  id: string;
+  a: boolean;
+  f: boolean;
+}
+
+export interface ICoastalBorder {
+  id: string;
+  a: boolean;
+  f: boolean;
+}
```

Comment:
aren't IBorder and ICoastalBorder the same? do we need two interfaces for these?